src/components/Card.tsx

```diff
@@ -1,18 +0,0 @@
-import React from "react";
-import { CardData } from "../model";
-type Props = CardData & { id: number; onFlip: (matchId: number) => void };
-
-export default class Card extends React.Component<Props> {
-  render() {
-    return (
-      <div onClick={() => this.props.onFlip(this.props.id)} className="scene">
-        <div className={`card ${!this.props.fold ? "flipped" : ""}`}>
```

Comment:
react class object

Reply:
https://stackoverflow.com/questions/34521797/how-to-add-multiple-classes-to-a-reactjs-component